test/plugins/plugins.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import unicode_literals
+
+import logging
+
+
+logger = logging.getLogger('temboardagent.' + __name__)
+
+
+class Hello(object):
+    def __init__(self):
+        logger.info("hello code running. Over.")
```

Comment:
This a dummy plugin example, right ? Could you rename the file to somehow more explicit ?